### ISSUE TITLE
Conda Package

### ISSUE THREAD

**Issue by curtisma** [niftylab/laygo2#3]
Conda is a package manager commonly used in the Python and data science community.
Would it be ok if I submitted a Conda recipe to [conda-forge](https://conda-forge.org/#page-top) to provide laygo2 as a Conda package?

If so, would anyone like to be included as another maintainer?  The CI /CD bot will update the Conda package with a PR every time a new PyPi package is released.  Then, usually, one of the maintainers just needs to merge the PR which has the updated version number and hash.

**Comment by jdhan** [comment]
Hi Curtis,

Laygo2 has submitted to PyPI already: https://pypi.org/project/laygo2/. Do
you think we need to submit it to conda-forge as well?

Best,

Jaeduk

2022년 9월 2일 (금) 오전 3:47, Curtis Mayberry ***@***.***>님이 작성:

**Comment by curtisma** [comment]
Yeah, for Conda users, it is best to install it as a conda package.  While you can install PyPi packages into a conda environment, it can lead to the PyPI dependencies overwriting existing versions of the dependencies.  Hence it's better to install packages into a conda environment as conda packages with Conda handling the dependencies.  Conda will also install the required Python interpreter into the environment.  I can add some basic instructions to the docs about installing it using Conda.

**Comment by jdhan** [comment]
Thanks for your clarification! Feel free to submit laygo2 as a conda package. Please assign me as one of the owners so that I can also make updates as needed.
